scripts/watcher_fsevents_cgo.go.patch

```diff
@@ -0,0 +1,22 @@
+--- /home/joel/Downloads/watcher_fsevents_cgo.go	2019-03-20 13:14:12.643748593 +1300
```

Comment:
Is this file related to cross compilation?

Reply:
Yes, at least until we upgrade our deps and Go. This patch is required for compiling for OSX and gets automatically applied by the build script.

Reply:
Thanks, that clarifies